.github/workflows/release.yml

```
@@ -19,7 +19,7 @@ jobs:
       # The DOCKER_IMAGE_TAG var is generated here from the release tag, after some validation
       - name: Generate docker image version from git tag
         run: |
-          echo "${{ github.event.release.tag_name }}" | grep -E '^[0-9.]*[0-9]$'
+          echo "${{ github.event.release.tag_name }}" | grep -E '^[v]*[0-9.]*[0-9]$'
```

```suggestion
          echo "${{ github.event.release.tag_name }}" | grep -E '^[v]?[0-9.]*[0-9]$'
```
We do not want it to match `vvvvv0.0.1`, right?